pkg/kubelet/apis/kubeletconfig/validation/validation_windows.go

```diff
@@ -0,0 +1,41 @@
+// +build windows
+
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation
+
+import (
+	"fmt"
+
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/kubernetes/pkg/kubelet/apis/kubeletconfig"
+)
+
+// ValidatePlatformConfiguration validates platform specific configuration and returns an error if it is invalid.
+func ValidatePlatformConfiguration(kc *kubeletconfig.KubeletConfiguration) error {
+	allErrors := []error{}
+
+	if kc.CgroupsPerQOS {
+		allErrors = append(allErrors, fmt.Errorf("invalid configuration: CgroupsPerQOS (--cgroups-per-qos) %v is not supported on Windows", kc.CgroupsPerQOS))
```

Comment:
Have a common formatted string for these two checking? Like

```
formatStr := "invalid configuration: %s %v is not supported on Windows"
```

Reply:
ACK